src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -208,88 +208,88 @@ <h1 class="infil">0.0 cm</h1>
 
                 <ul id="thumbs-land" class="nav" role="tablist"> <!-- Landcover Thumbs -->
 
-                    <li id="thumb-li_residential" class="active"> <!-- LIR Thumb -->
-                        <a href="#land-li_residential" aria-controls="filter-tab" role="tab" data-toggle="tab">
-                            <img src="{% static 'images/water_balance/thumb_lir.png' %}" alt="Low-Intensity Residential" data-container="body" data-toggle="popover" data-placement="left" data-title="Low Intensity Residential" data-content="Areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 20% to 49% percent of total cover. These areas most commonly include single-family housing units.">
+                    <li id="thumb-open_water">
+                        <a href="#land-open_water" aria-controls="filter-tab" role="tab" data-toggle="tab">
+                            <img src="" style="background-color: gray;" alt="Open Water" data-container="body" data-toggle="popover" data-placement="left" data-title="Open Water" data-content="Areas of open water, generally with less than 25% cover of vegetation or soil.">
                         </a>
-                        <label>LIR</label>
+                        <label>Water</label>
                     </li>
 
-                    <li id="thumb-hi_residential"> <!-- HIR Thumb -->
-                        <a href="#land-hi_residential" aria-controls="filter-tab" role="tab" data-toggle="tab">
-                            <img src="{% static 'images/water_balance/thumb_hir.png' %}" alt="High-Intensity Residential" data-container="body" data-toggle="popover" data-placement="left" data-title="High Intensity Residential" data-content="Areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 50% to 79% of the total cover. These areas most commonly include single-family housing units.">
+                    <li id="thumb-developed_open">
+                        <a href="#land-developed_open" aria-controls="filter-tab" role="tab" data-toggle="tab">
+                            <img src="{% static 'images/water_balance/thumb_turfGrass.png' %}" alt="Developed, Open Space" data-container="body" data-toggle="popover" data-placement="left" data-title="Developed, Open Space" data-content="Areas with a mixture of some constructed materials, but mostly vegetation in the form of lawn grasses. Impervious surfaces account for less than 20% of total cover. These areas most commonly include large-lot single-family housing units, parks, golf courses, and vegetation planted in developed settings for recreation, erosion control, or aesthetic purposes.">
                         </a>
-                        <label>HIR</label>
+                        <label>Developed, Open Space</label>
```

Comment:
Actually, I just revisited the original issue, and while they do give acceptable shortened versions for "Developed, Low (Medium and High) Intensity" they don't give once for open space. So in the interest of sticking to the request as written, I'll leave this. It's easy to revise later and we may end up shortening more of these labels so we can do them all at once.